DeviceAdministration/Infrastructure.UnitTests/Simulator.WebJob/JsonSerializeTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WebJob.SimulatorCore.Serialization;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.UnitTests.Simulator.WebJob
+{
+    class JsonSerializeTests
+    {
+        private JsonSerialize serializer;
```

Comment:
this class has 2 methods: Serialize and Deserialize and they both have 1 references, we can just inline the methods and remove this class ?